### PR TITLE
DS-2034 Introduce guidelines about deploying a new release to RubyGems

### PR DESCRIPTION
**Pull request by ildomar-grings**
Introduce a long explanation about how to deploy a new Gem release to Rubygems, using GitHub UI, Travis CI, and its triggers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,39 @@ If working on an installed copy of the gem through normal channels, you should b
...
+We imagine the scenario where we will increase the Hoodoo version, so this will be the expected sequence of steps:
+
+1. Update version file, `lib/hoodoo/version.rb`, increasing constants VERSION and DATE
+2. Update Readme file itself, topic `Note`, to contain the expected upcoming version
+3. Run the command `bundle install`. Doing so, Bundle will be updating the file `Gemfile.lock` with the upcoming version
+4. Start a new feature branch
+5. Commit changes  
+6. Create a Pool Request
+7. Get approval for this Pool Request 
+8. Merge the Pool Request
```
Comment (nickpiercy95) [inline README.md:86]: `Pool` -> `Pull`

File: README.md [hunk 2]
```diff
@@ -62,6 +62,39 @@ If working on an installed copy of the gem through normal channels, you should b
...
+9. Now comes the tricky part. Until now, wasn't asked to do any kind of tagging, which is ok. This step can be achieved using the proper GitHub UI, we just need to follow some very simple steps. 
+   1. Access Hoodoo's repository home page, `https://github.com/LoyaltyNZ/hoodoo`
+   2. Then access the list of existent releases, clicking on `Releases` link (https://github.com/LoyaltyNZ/hoodoo/releases)
+   3. Now we need to start a Draft, clicking on `Draft a new release`
+   4. Type a title and description
+   5. Click on `Choose a tag`, and type a new one, following the expected standards. Please do not select any existent tag, they do belong to older releases.
```
Comment (nickpiercy95) [inline README.md:92]: Should we say exactly what the expected standards are? In fact, would it be easiest just to say "copy over the entry from the changelog"?

File: README.md [hunk 3]
```diff
@@ -88,6 +121,7 @@ Some additional higher level hand written documentation may also be present as M
 - [Patrick Copeland](https://github.com/pjscopeland)
 - [Rory Stephenson](https://github.com/thelollies)
 - [Wayne Hoover](https://github.com/waynehoover)
+- [Ildomar Grings](https://github.com/ildomar-grings)
```
Comment (nickpiercy95) [inline README.md:124]: 😎

**Review by nickpiercy95 — CHANGES_REQUESTED** [review]
Looks good Ildo 👍, just a couple of minor comments

File: README.md (hunk 1 above)
Comment (ildomar-grings) [reply README.md:86]: Oh boy, that was funny. Way too many movies in my head.

File: README.md (hunk 2 above)
Comment (ildomar-grings) [reply README.md:92]: Good point, I will copy it

File: README.md (hunk 1 above)
Comment (ildomar-grings) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (ildomar-grings) [reply README.md]: Done

**Review by nickpiercy95 — APPROVED** [review]

**Pull request merged**